src/components/Search/Search.jsx

```diff
@@ -61,25 +62,31 @@ class Search extends Component {
     }
 
     componentDidMount() {
-        /// Get relevent query results from api
-        axios({
-            method: 'get',
-            url: `${BASE_API_PATH}/assets/search?q=${this.state.query}`,
-            headers: { 'content-type': 'application/json' },
-            timeout: API_TIMEOUT
-        }).then(result => {
-            this.setState({
-                results: result.data.assets,
-                loaded: true,
+        if ( Session.isSignedIn() ) {
+            /// Get relevent query results from api
+            axios({
+                method: 'get',
+                url: `${BASE_API_PATH}/assets/search?q=${this.state.query}`,
```

Comment:
Use the helperFile.js methods

Reply:
> Just change those url's to use the helperFile.js methods and it's good. You should be able to pass the data needed for those url's through as function parameter 😉


4cfafd6 Added the changes you requested to the helperFile/frontend code